packages/react/src/internal/ComponentToggle.js

```diff
@@ -9,22 +9,19 @@ import React from 'react';
 import { useFeatureFlag } from '../components/FeatureFlags';
 
 export function createComponentToggle(spec) {
-  const { name, flag = 'enable-v11-release', next, classic } = spec;
+  const { flag = 'enable-v11-release', next, classic } = spec;
 
   function ComponentToggle(props, ref) {
     const enabled = useFeatureFlag(flag);
     if (enabled) {
       if (next) {
         return React.createElement(next, { ...props, ref: ref });
-      } else {
-        return null;
       }
+      return null;
     }
 
     return React.createElement(classic, { ...props, ref: ref });
   }
 
-  ComponentToggle.displayName = `FeatureToggle(${name})`;
-
   return React.forwardRef(ComponentToggle);
```

Comment:
Would it work for the snapshot and the devtools if we applied the displayName to the result of the forwardRef call?

```js
// not sure if this one would still be needed
ComponentToggle.displayName = `FeatureToggle(${name})`;

const wrapped = React.forwardRef(ComponentToggle);
wrapped.displayName = `FeatureToggle(${name})`;
return wrapped;
```

If the function name has to be dynamic, it looks like function names are configurable, but not writable. Would maybe need to be something like below?
```js
Object.defineProperty(ComponentToggle, 'name', {value: `FeatureToggle(${name})`, configurable: true});

return React.forwardRef(ComponentToggle);
```

Reply:
yeah I think the first codeblock should work (and we can still remove the original `ComponentToggle.displayName` assignment). I'll test it out